daemon/remote/cmd/cmuxd-remote/hooks.go

```
@@ -0,0 +1,209 @@
+		return
+	}
+	tty, openErr := os.OpenFile("/dev/tty", os.O_WRONLY, 0)
+	if openErr != nil {
+		return
+	}
+	defer tty.Close()
+	if os.Getenv("TMUX") != "" {
+		// Passthrough must be on for tmux to forward the wrapped sequence;
+		// the wrapper enables it at launch, but a tmux server restarted since
+		// then would have lost it.
+		_ = exec.Command("tmux", "set", "-g", "allow-passthrough", "on").Run()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)